internal/builders/generic/README.md

```diff
@@ -64,6 +64,8 @@ output:
 $ sha256sum artifact1 artifact2 ... | base64 -w0
 ```
 
+This workflow expects the base64 encoded value for the subjects to decode to a string conforming to the expected output of the above command. Specifically, the decoded output is expected to be comprised of a hash value followed by a space followed by the artifact name.
```

Comment:
Technically the above command outputs the base64 value so the value of `base64-subjects` wouldn't wouldn't decode to that. It would decode to the value of the `sha256sum artifact1 artifact2 ...` part. Rather than "above command". I'd just say sha256sum command.

Also consider referring to the input name directly so users know exactly what we're talking about.
e.g. "This workflow expects the value of `base64-subjects to ..."
```suggestion
This workflow expects the `base64-subjects` input to decode to a string conforming to the expected output of the `sha256sum` command. Specifically, the decoded output is expected to be comprised of a hash value followed by a space followed by the artifact name.
```

Reply:
Additionally, consider adding this as a comment in the example yaml below.
```yaml
with:
  # This workflow expects the value of base64-subjects to decode to a string conforming to the
  # expected output of the sha256sum command...
  base64-subjects: ...